src/meta/meta_bulk_load_service.cpp

```diff
@@ -1181,11 +1169,7 @@ bool bulk_load_service::check_ever_ingestion_succeed(const partition_configurati
 // ThreadPool: THREAD_POOL_META_STATE
 void bulk_load_service::partition_ingestion(const std::string &app_name, const gpid &pid)
 {
-    FAIL_POINT_INJECT_F("meta_bulk_load_partition_ingestion", [=](dsn::string_view) {
-        if (_apps_ingesting_count[pid.get_app_id()] < FLAGS_bulk_load_ingestion_concurrent_count) {
-            _apps_ingesting_count[pid.get_app_id()]++;
-        }
-    });
+    FAIL_POINT_INJECT_F("meta_bulk_load_partition_ingestion", [](dsn::string_view) {});
```

Comment:
why keep an empty function here?

Reply:
This is the test mock function. In unit test, function `partition_ingestion` sometimes need to be skipped.